roles/redpanda_broker/defaults/main.yml

```diff
@@ -23,3 +23,6 @@ redpanda_key_file: "{{ redpanda_certs_dir }}/node.key"
 redpanda_cert_file: "{{ redpanda_certs_dir }}/node.crt"
 redpanda_truststore_file: "{{ redpanda_certs_dir }}/truststore.pem"
 node_exporter_version: "{{ node_exporter_custom_version | default('1.5.0') }}"
+cloud_storage_credentials_source: "aws_instance_metadata"
+cloud_storage_enable_remote_write: "true"
+cloud_storage_enable_remote_read: "true"
```

Comment:
@twmb these are defaults for the template, but are not enabled by default because 

``` yaml
cloud_storage_enabled: {{ true if tiered_storage_bucket_name is defined and tiered_storage_bucket_name|d('')|length > 0 else false }}
``` 

cloud_storage_enabled is false by default. I've set them to true because I feel like the likely assumption for a new user who is enabling tiered storage is that their topics are using tiered storage by default. It's straightforward to disable so they can always turn it off if they want to explicitly create tiered storage enabled topics.

I pulled cloud_storage_credentials_source into a variable and out of the template so that people can set it to gcp_instance_metadata. That _should_ be all that is required to make tiered storage work for GCP.

Reply:
tagging @hcoyote to confirm, this makes sense to me

Reply:
for cloud_storage_credentials_source: I do wonder if we should default to config_file for this to match what redpanda default is.

there's a bit of magic going on with using aws/gcp instance_metadata in that you have to externally wire up the iam role ... and we can only use instance_metadata for cloud-based installs, so defaulting to config_file seems like the lowest common denominator -- see notes on  https://docs.redpanda.com/docs/manage/security/iam-roles/

Reply:
And thinking about the remote_read/remote_write settings: I'm not sure yet if we should be enabling this by default for all topics on the cluster or letting the user set this on a per-topic basis.  I could be convinced of either direction. I wonder if we're better of having a different variable that explicitly talks about setting whole cluster offloading, that sets these two on the user's behalf. I can't think of a situation really where we'd not be setting both of these to true or both false. Maybe in the case where you're doing a topic recovery on a new cluster, you'd only want to set remote_read?